npm/packages/@socketsupply/socket/package.json

```diff
@@ -5,10 +5,6 @@
   "type": "module",
   "main": "index.js",
   "types": "index.d.ts",
-  "exports": {
```

Comment:
can we keep this as:

```
  "exports": {
    "types": "./index.d.ts"
  },
```

Reply:
im not sure if there is a way, specifying the `exports` field is the problem because iirc it makes it exclusive.